### PR TITLE
Update community-process.rst

### PR DESCRIPTION
**Pull request by lindy65**
Included UserVoice Issues process

### CODE REVIEW

File: source/process/community-process.rst [hunk 1]
```diff
@@ -54,8 +54,31 @@ The various community channels are outlined below, along with the responsibiliti
...
+
+        - Thank you [NAME] for nominating the feature! It is available in Mattermost [X.Y]. Would anyone like to help re-tweet the announcement? [LINK TO TWEET]
+
+    - Sales Responsibility - Enterprise Edition:
+    
+      - Responds to the issues and close them off after release date.
+        
+      - `Spreadsheet <https://docs.google.com/spreadsheets/d/1nljd4cFh-9MXF4DxlUnC8b6bdqijkvi8KHquOmK8M6E/edit#gid=0>`_ to track UserVoice issues to be announced in upcoming releases.
+
+3. `Community Channel <https://pre-release.mattermost.com/core/channels/community>`_: GitHub Issues with Activity Older than 7 days that have not been created by Mattermost staff
```
Comment (jasonblais) [inline source/process/community-process.rst]: I think these go to Contributors channel too

File: source/process/community-process.rst [hunk 2]
```diff
@@ -54,8 +54,31 @@ The various community channels are outlined below, along with the responsibiliti
...
+      
+    - Community PM Responsibility - Team Edition:
+    
+      - The PM who is release manager for the current release cycle will update the issues after release date.
+
+        - Thank you [NAME] for nominating the feature! It is available in Mattermost [X.Y]. Would anyone like to help re-tweet the announcement? [LINK TO TWEET]
+
+    - Sales Responsibility - Enterprise Edition:
+    
+      - Responds to the issues and close them off after release date.
```
Comment (jasonblais) [inline source/process/community-process.rst]: close --> closes

is 'off' needed?

File: source/process/community-process.rst [hunk 3]
```diff
@@ -54,8 +54,31 @@ The various community channels are outlined below, along with the responsibiliti
   - Community PM Responsibility:
 
     - Review the list for posts that haven’t been answered, and either reply (with help from dev if needed) or route to the appropriate person, following up to make sure a response is posted
+    
+2. `Contributors Channel <https://pre-release.mattermost.com/core/channels/tickets>`_: UserVoice Issues
 
-2. `Community Channel <https://pre-release.mattermost.com/core/channels/community>`_: GitHub Issues with Activity Older than 7 days
+  - Description:
+
+    - Ops posts at-mentioning the Community PM with a list that includes issues:
```
Comment (esethna) [inline source/process/community-process.rst]: Propose at-mention > @mention

File: source/process/community-process.rst [hunk 4]
```diff
@@ -54,8 +54,31 @@ The various community channels are outlined below, along with the responsibiliti
...
+    
+      - The PM who is release manager for the current release cycle will update the issues after release date.
+
+        - Thank you [NAME] for nominating the feature! It is available in Mattermost [X.Y]. Would anyone like to help re-tweet the announcement? [LINK TO TWEET]
+
+    - Sales Responsibility - Enterprise Edition:
+    
+      - Responds to the issues and close them off after release date.
+        
+      - `Spreadsheet <https://docs.google.com/spreadsheets/d/1nljd4cFh-9MXF4DxlUnC8b6bdqijkvi8KHquOmK8M6E/edit#gid=0>`_ to track UserVoice issues to be announced in upcoming releases.
```
Comment (esethna) [inline source/process/community-process.rst]: I would add "Updates" to the beginning of this bullet to be more consistent

File: source/process/community-process.rst (hunk 2 above)
Comment (lindy65) [inline source/process/community-process.rst]: Agree, updated.

File: source/process/community-process.rst (hunk 1 above)
Comment (lindy65) [inline source/process/community-process.rst]: Good catch!  Thanks, updated.

File: source/process/community-process.rst (hunk 3 above)
Comment (lindy65) [inline source/process/community-process.rst]: Updated to match line 85 (for uniformity)

File: source/process/community-process.rst (hunk 4 above)
Comment (lindy65) [inline source/process/community-process.rst]: Thanks, updated.

**Pull request merged**